test/e2e/pages/move-detail.js

```diff
@@ -79,7 +82,7 @@ class MoveDetailPage extends Page {
             dateOfBirth
           )})`
         : undefined,
-      Gender: gender,
+      // Gender: gender,
```

Comment:
Rogue comment?

Reply:
It was failing. But now it's not so reinstated.